README.md

```diff
@@ -91,33 +161,86 @@ client
 
-If you want to use RxJava instead, call the `observe()` method to get an `Observable` instance:
 ```java
 client
     .observe(CDAAsset.class)
     .one("jake")
     .subscribe(System.out::println);
 ```
 
-### Default Ordering
+Paging
+------
+
+If more then _100_ Resources are in the Space, `.fetchAll()` will only return the first _100_. If more Resources are needed, specify the limit with the `.limit(X)` like so:
+
+```java
+CDAArray result = 
+  client
+    .fetch(CDAEntry.class)
+    .limit(1000)
+    .all();
+```
+
+The maximum number of Resources to be requested is _1000_. 
 
-Bear in mind that there is no default ordering included for any method which returns a `CDAArray` instance. This means that if you plan to page through more than 100 results with multiple requests, there is no guarantee that you will cover all entries. It is however possible to specify custom ordering:
+For more then _1000_ Resources `.skip(N)`, `.limit(L)` and `.orderBy(F)` methods are needed. By using `.skip(N)`, the first _N_ Resources are ignored and _L_, from `.limit(L)`, items are returned. 
+
+To guarantee ordering, the use of the `.orderBy` method is required: It enforces the _Array_<sup><a href="https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/collection-resources-and-pagination" title="A collection of Resources from Contentful. Containts meta information about number and limits of the Resources it contains."/>i</a></sup> to be in a predictable order. 
+
+The following code can be used to request all Entries:
+
+```java
+// get the amount of Entries, without fetching the actual content
+final int amountOfResourcesInContentful = 
+  client
+    .fetch(CDAEntry.class)
+    .limit(0)
+    .all()
+    .total();
+
+// create storage for the Entries
+final List<CDAResource> resources = new ArrayList<CDAResource>(amountOfResourcesInContentful);
+
+// use page size, based on usecase
+final int PAGE_SIZE = 2;
+
+// loop through all pages and store results
+for(int page = 0; page * PAGE_SIZE < amountOfResourcesInContentful; ++page) {
+  final CDAArray currentPagedItems = client
+      .fetch(CDAEntry.class)
+      .skip(page * PAGE_SIZE)
+      .limit(PAGE_SIZE)
+      .orderBy("sys.createdAt")
+      .all();
+
+  // add to current list of Entries
+  resources.addAll(currentPagedItems.items());
+}
+```
+
+Using the `.reverseOrderBy()` method reverses the order:
 
 ```java
 CDAArray result = 
     client
         .fetch(CDAEntry.class)
+        .limit(23)
         .reverseOrderBy("sys.createdAt")
         .all();
 ```
 
-The above snippet will fetch all Entries, ordered by newest-to-oldest.
+The above snippet will fetch the first _23_ Entries, ordered by newest-to-oldest creation day.
```

Comment:
|Before|After|
|------|-------|
|ordered by newest-to-oldest creation day| sorted by creation date with the latest ones on top.|